### PR TITLE
Fix select, checkbox and options filter to use field name and value

### PR DESCRIPTION
**Pull request by ShoboySnr**

### CODE REVIEW

File: src/OptinForms/AbstractOptinTheme.php [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ public function shortcode_optin_form_custom_fields($atts)
 
                     $options = [];
                     if ( ! empty($field['field_options'])) {
-                        $options = array_map('trim', explode(',', $field['field_options']));
+                        $options = preg_split( "/\\r\\n|\\r|\\n/", $field['field_options']);
```
Comment (w3guy) [inline src/OptinForms/AbstractOptinTheme.php:1052]: @ShoboySnr hey.. looks like you do not think of old or existing users when making changes. This is not good

what happens to customer already using comma to separate the options?

To maintain backward compatibility, I prefer something like

Name|f_name, Email|email

If no pipe is found, default the entered data to be the option name and value. Just like the old behaviour.

File: src/Admin/Customizer/CustomControls/WP_Customize_Fields_Repeater_Control.php [hunk 2]
```diff
@@ -659,7 +659,7 @@ public function template($index = 9999999999999)
                     <?php $this->parse_control($index, apply_filters('mo_optin_fields_controls_before', [], $this->optin_campaign_id, $index, $this->saved_values)); ?>
                     <?php $this->repeater_text_field($index, 'placeholder', '', __('Title', 'mailoptin')); ?>
                     <?php $this->repeater_select_field($index, 'field_type', $field_types, '', __('Type', 'mailoptin')); ?>
-                    <?php $this->repeater_textarea_field($index, 'field_options', '', __('Options', 'mailoptin'), __('Enter a comma-separated list of options', 'mailoptin')); ?>
+                    <?php $this->repeater_textarea_field($index, 'field_options', '', __('Options', 'mailoptin'), __('Enter each option in a separate line. To differentiate between label and value, separate them with a pipe char ("|"). For example: First Name|f_name', 'mailoptin')); ?>
```
Comment (w3guy) [inline src/Admin/Customizer/CustomControls/WP_Customize_Fields_Repeater_Control.php:662]: @ShoboySnr when u done with the change i mentioned earlier, correct this description

**Review by w3guy — CHANGES_REQUESTED** [review]

File: src/OptinForms/AbstractOptinTheme.php (hunk 1 above)
Comment (ShoboySnr) [reply src/OptinForms/AbstractOptinTheme.php:1052]: Ohhh.. Okay. Got it. I'll effect the change immediately

File: src/Admin/Customizer/CustomControls/WP_Customize_Fields_Repeater_Control.php (hunk 2 above)
Comment (ShoboySnr) [reply src/Admin/Customizer/CustomControls/WP_Customize_Fields_Repeater_Control.php:662]: yeah I'll

**Pull request merged**